### PR TITLE
Calimero update from 2.2.0-SNAHSHOT to 2.2.1

### PR DESCRIPTION
**Pull request by Snickermicker**
KNXCoreTypeMapper and respective tests updated to include new KNX data
point types.
#3264
Signed-off-by:  <vd-github@online.de> (github: Snickermicker)

### CODE REVIEW

File: bundles/binding/org.openhab.binding.knx/.classpath [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	<classpathentry kind="src" output="target/classes" path="src/main/java"/>
 	<classpathentry excluding="**" kind="src" output="target/classes" path="src/main/resources"/>
 	<classpathentry exported="true" kind="lib" path="lib/calimero-rxtx-2.0a3.jar"/>
-	<classpathentry kind="lib" path="lib/calimero-core-2.2.0-SNAPSHOT.jar"/>
+	<classpathentry kind="lib" path="lib/calimero-core-2.2.1.jar" sourcepath="/home/vdaube/development/git/calimero-core/target/calimero-core-2.2.1-sources.jar"/>
```
Comment (watou) [inline bundles/binding/org.openhab.binding.knx/.classpath]: remove sourcepath="..."

**Comment by watou** [comment]
@Snickermicker, could you describe the value of this upgrade, identify if it fixes a defect users are experiencing with 1.8.0 (or is it an enhancement), address my previous comment, rebase this PR off current master, share a test JAR with other KNX binding users to validate its correctness, incorporate their feedback as appropriate, and squash your commits?  Thanks very much.

**Comment by teichsta** [comment]
@Snickermicker why did you close this PR? new KNX DPTs seem to be a valid reason to change to a new lib version?

**Comment by Snickermicker** [comment]
@teichsta I too believe it is a valid reason. Unfortunately, I've only very limited time to work on openhab (which I like very much). I don't have the time to fulfill watou's (valid) request to "share a test JAR with other KNX binding users to validate its correctness, incorporate their feedback as appropriate".
So, instead of leaving this pull request open and stop other contributors from adding this functionality, I thought I close it.

**Pull request closed without merging**